gym/lib/gym/options.rb

```diff
@@ -123,6 +123,14 @@ def self.plain_options
                                      conflict_block: proc do |value|
                                        UI.user_error!("'#{value.key}' must be false to use 'export_options'")
                                      end),
+        FastlaneCore::ConfigItem.new(key: :export_xcargs,
+                                     env_name: "GYM_EXPORT_XCARGS",
+                                     description: "Pass additional arguments to xcodebuild for the export phase. Be sure to quote the setting names and values e.g. OTHER_LDFLAGS=\"-ObjC -lstdc++\"",
```

Comment:
Please also update the regular `xcargs` action description to reflect that that only applies to the build phase.